datasets/c3/README.md

```diff
@@ -6,7 +6,7 @@ language_creators:
 language:
 - zh
 license:
-- other-non-commercial-research
+- other
```

Comment:
When removing the text after "other", shouldn't we add that information with the `license-details` tag (like done e.g. in "afrikaans_ner_corpus"), to avoid losing that information?

In this particular example, I think it is important to keep that the license is only for non-commercial research purpose.

Reply:
Custom licenses can be written in the `### Licensing Information` section I think.

I think we can re-add this info in a subsequent PR